src/display.rs

```diff
@@ -109,25 +134,44 @@ impl<'a, S: 'a + ToOwned + ?Sized> AnsiGenericString<'a, S>
 where
     <S as ToOwned>::Owned: fmt::Debug,
 {
+    /// Directly access the style
+    pub const fn style_ref(&self) -> &Style {
+        &self.style
+    }
+
+    /// Directly access the style mutably
+    pub fn style_ref_mut(&mut self) -> &mut Style {
+        &mut self.style
+    }
+
+    // Directly access the underlying string
+    pub fn as_str(&self) -> &S {
+        self.string.as_ref()
+    }
```

Comment:
Can you make this a doc comment?

```suggestion
    /// Directly access the underlying string
    pub fn as_str(&self) -> &S {
        self.string.as_ref()
    }
```

(I know you just moved this up here, but worth the cleanup)

Reply:
@sholderbach 
> Can you make this a doc comment?

Done.

Reply:
Thx!